### ISSUE TITLE
Vibration has stopped working in "Dinput only" mode

### ISSUE THREAD

**Issue by cadegrey** [Ryochan7/DS4Windows#429]
Running Windows 10 Pro.

A few days ago the vibration was working just fine, but now no games seem to be working with it. Clearly the vibration works, as "Test Heavy" and "Test Light" are showing that they are. I've tested it with both A Hat in Time as well as Rocket League to make sure it wasn't the games, it's definitely an issue with DS4Windows.

As an aside, minimize to taskbar is not working either,  just seems to minimize normally (without going to the taskbar). It could be related for all I know, but I'm not sure.

**Comment by Ryochan7** [comment]
Does the DS4 even support force feedback when using DirectInput? I tried the DS4 (USB) in Rocket League without DS4Windows and a fresh connection and rumble did not work at all.

As for the minimize to taskbar option, is the option enabled in the Settings tab? It sounds like the program is minimizing to the system tray. Minimize to taskbar is disabled by default. Minimize to taskbar works for me when the option is enabled.

**Comment by cadegrey** [comment]
I remember the rumble working with DirectInput at some point, don't know when it stopped working exactly because I wasn't on my PC for a very long time.

The DS4 doesn't support force feedback when not using DS4Windows, only when DS4Windows is open it does. I've checked on many forums just to be sure. I should mention that I have 2 different profiles for DS4Windows, one that is just the default Xinput mode, and one that just has "Use Dinput only" enabled so games (that support it) will show the DS4 buttons.

And oh my God, I just tested again, I am very sorry. I got the options completely confused on the minimize to taskbar option. No issue there, now.
